Modules/posixmodule.c

```diff
@@ -416,7 +417,6 @@ extern char        *ctermid_r(char *);
 #  ifndef IO_REPARSE_TAG_MOUNT_POINT
 #    define IO_REPARSE_TAG_MOUNT_POINT (0xA0000003L)
 #  endif
-#  include "osdefs.h"             // SEP
```

Comment:
Additionally:

* `IO_REPARSE_TAG_MOUNT_POINT` and `IO_REPARSE_TAG_SYMLINK` are WinAPI constants. 
* `HAVE_SYMLINK` doesn't depend on the compiler. (For some reason it was decided to define `HAVE_SYMLINK` but not `HAVE_READLINK`.)
* "windows.h" is already included. 
* "lmcons.h" (LAN Manager constants) and "shellapi.h" are Windows headers. There's no reason to include "shellapi.h" since `os.startfile()` uses common WinAPI types and links to `ShellExecuteW()` at runtime via `GetProcAddress()`.

Reply:
Noted, should I simply just remove the IO_REPARSE macros entirely in that case?

Reply:
> Noted, should I simply just remove the IO_REPARSE macros entirely in that case?

Yes, note that `IsReparseTagNameSurrogate()` is used unconditionally in `win32_xstat_impl()`. This macro is defined in the same section of "winnt.h" that defines `IO_REPARSE_TAG_MOUNT_POINT` and `IO_REPARSE_TAG_SYMLINK`.

Reply:
Removed macros as requested